gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-account/src/main/java/io/gravitee/am/gateway/handler/account/resources/account/AccountResponseHandler.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.am.gateway.handler.account.resources.account;
+
+import io.gravitee.am.model.Factor;
+import io.gravitee.am.model.common.Page;
+import io.gravitee.am.reporter.api.audit.model.Audit;
+import io.gravitee.common.http.HttpHeaders;
+import io.gravitee.common.http.MediaType;
+import io.vertx.core.http.HttpClientRequest;
+import io.vertx.core.json.Json;
+import io.vertx.reactivex.core.http.HttpServerResponse;
+import io.vertx.reactivex.ext.web.RoutingContext;
+
+import java.util.List;
+
+public class AccountResponseHandler {
+
+    private static HttpServerResponse buildDefaultHeader(RoutingContext routingContext) {
+        return routingContext.response()
+                .putHeader(HttpHeaders.CACHE_CONTROL, "no-store")
+                .putHeader(HttpHeaders.PRAGMA, "no-cache")
+                .putHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON);
+    }
+
+    public static void handleWIP(RoutingContext routingContext) {
+        buildDefaultHeader(routingContext).end("{ \"temp\" : true }");
+    }
+
+    public static void handleDefaultResponse(RoutingContext routingContext, Object obj){
+        buildDefaultHeader(routingContext).end(Json.encodePrettily(obj));
+    }
+
+    public static void handleUpdateUserResponse(RoutingContext routingContext) {
+        buildDefaultHeader(routingContext).end("{ \"status\" : \"OK\" }");
```

Comment:
Instead of create JSON directly as a String, I think it is more maintainable to have an object that will be serialized into JSON format. (same comment for line 56)